cmd/root.go

```diff
@@ -36,6 +36,11 @@ func ServerFlags(flags *pflag.FlagSet) {
 	flags.StringSliceVar(&pullServers, "pull-servers", []string{}, "push check results to multiple canary servers")
 	flags.StringVar(&runner.RunnerName, "name", "local", "Server name shown in aggregate dashboard")
 	flags.StringVar(&prometheusURL, "prometheus", "", "URL of the prometheus server that is scraping this instance")
+	flags.StringVar(&cache.PostgresHost, "postgres-host", "", "Address for the postgres database")
```

Comment:
I think we just want to use a connection string with a default of CACHE_CONNECTION_STRING env var